src/components/media-filter-control/styles/editor.scss

```diff
@@ -0,0 +1,26 @@
+.components-coblocks-media-filter {
+
+	&.has-filter {
+		position: relative;
+
+		::after {
```

Comment:
Is this working? I was sure you needed `&::after {`

Reply:
It adds the little triangle that we use for the animations control as well - when an image filter is added.

Reply:
But the `&` isn't required? Good to know!